### PR TITLE
support put operation

### PR DESCRIPTION
**Pull request by WeiXinChan**
## Summary

- support put operation

- -put means override a record with current record.
- -can not use put operation when table has any index.

### CODE REVIEW

File: src/test/java/com/alipay/oceanbase/rpc/ObTablePutTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    @Before
+    public void setup() throws Exception {
+        final ObTableClient obTableClient = ObTableClientTestUtil.newTestClient();
+        obTableClient.init();
+        this.client = obTableClient;
+        client.addRowKeyElement(tableName, new String[] { "id" });
+    }
+
+    /**
+     CREATE TABLE IF NOT EXISTS `test_put` (
```
Comment (IHEII) [inline src/test/java/com/alipay/oceanbase/rpc/ObTablePutTest.java:48]: Should we try lob in the test?

File: src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/ObTableOperationType.java [hunk 2]
```diff
@@ -26,7 +26,10 @@ public enum ObTableOperationType {
     INSERT_OR_UPDATE(4), // INSERT or UPDATE, columns not in arguments will remain unchanged
     REPLACE(5), // DELETE & INSERT, columns not in arguments will change to default value
     INCREMENT(6), // the column must be can be cast to long. if exist increase, else  insert
-    APPEND(7);// append column value
+    APPEND(7),// append column value
+    SCAN(8),// scan value
+    TTL(9),// expire value
```
Comment (IHEII) [inline src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/ObTableOperationType.java:31]: What is the type of TTL operation type

File: src/test/java/com/alipay/oceanbase/rpc/ObTablePutTest.java (hunk 1 above)
Comment (WeiXinChan) [reply src/test/java/com/alipay/oceanbase/rpc/ObTablePutTest.java:48]: We will test before Pr (perf_opt to master)

File: src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/ObTableOperationType.java (hunk 2 above)
Comment (WeiXinChan) [reply src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/ObTableOperationType.java:31]: Here ObTableOperationType is same as definition of observer。TTL is an special operation in observer。

**Comment by IHEII** [comment]
LGTM

**Pull request merged**